README.md

```diff
@@ -92,6 +92,17 @@ become worse.
 
 # Changes
 
+* *Version 1.9.4* - Introduce Connection Name Validation
```

Comment:
Perhaps just "unreleased" since we have yet to push it out.

Reply:
Well, I cut a new release, since the 1.9.3 didn't build with Erlang 19.3